cmd/ndm_daemonset/probe/eventhandler.go

```diff
@@ -186,14 +195,6 @@ func generateUUID(bd blockdevice.BlockDevice) (string, bool) {
 		klog.Infof("device(%s) has a WWN, using WWN: %s", bd.DevPath, bd.DeviceAttributes.WWN)
 		uuidField = bd.DeviceAttributes.WWN
 		ok = true
-	case len(bd.PartitionInfo.PartitionTableUUID) > 0:
-		// PartitionTable UUID is used if the disk contains only the partition table and not a partition. This is a very rare
-		// case where the user creates the partition table, but does not create any partition on the disk. In such a case
-		// partition table UUID will be used. In all other cases where partition table UUID is present, the disk will also
-		// have partition entries and it will be used.
-		klog.Infof("device(%s) has a partition table, using partition table UUID: %s", bd.DevPath, bd.PartitionInfo.PartitionTableUUID)
-		uuidField = bd.PartitionInfo.PartitionTableUUID
-		ok = true
 	case len(bd.FSInfo.FileSystemUUID) > 0:
 		klog.Infof("device(%s) has a filesystem, using filesystem UUID: %s", bd.DevPath, bd.FSInfo.FileSystemUUID)
 		uuidField = bd.FSInfo.FileSystemUUID
```

Comment:
One more query @akhilerm, consider this case:

- User creates a GKE cluster with Local SSDs, that can formated and mounted by default. 
- User installs openebs
- NDM discovers the block devices using the FS ID
- Users wants to install cStor on those block devices, so user follows the troubleshooting guide and realizes that blockdevices should not have a file system. So "wipefs -a` is executed. 

What happens to those block devices? Do we end up creating stale block devices? Not as part of this PR, but should we considering a Block Device, that is unknown state but is not claimed? 

In the above scenario - it is possible that BDC was already attached to the Old BDs, but then due to user erasing the Filesystem, those BDs will go to unknown state.  Or I think cStor doesn't claim devices that already have filesystems on it, which is better.

Reply:
- cStor will pick up only devices which does not have a filesystem to create pools.

But the case that you mentioned can happen, to prevent it while creating BlockDeviceClaim, the user can specify the `VolumeMode` for the BlockDevice to claim. If VolumeMode is specified as `Block`, only devices without any filesystem will be claimed. 
Currently by default, if volume mode is not specified, any block device can be claimed. I think, if we change the default behaviour to claim only blockdevices in `Block` mode, this issue will not happen.